### PR TITLE
wag js client upgrade

### PR DESCRIPTION
**Pull request by ChrisScotMartin**
## Jira

[JIRA
](https://clever.atlassian.net/browse/INFRANG-6125)
## Testing

Created several wag services with this version of wag, imported their clients and verified propagation of baggage headers and logs from node -> wag thing -> wag thing

## Checklist
- [x] Run `make build`
- [x] Run `make generate`
- [x] Update the current version in the `/VERSION` file. First line should be JUST the version e.g. `v10.13.1`. Then a blank line. Then release notes.

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1,4 +1,6 @@
-v9.2.2
+v9.2.3
```
Comment (tnsardesai) [inline VERSION:1]: nit - https://semver.org/ With the new added functionality this should be a 9.3.0

File: samples/test/js/paging.ts [hunk 2]
```diff
@@ -21,6 +21,7 @@ describe("paging", function() {
       .query({startingAfter: "2"})
       .reply(200, [{id: 3, name: "third"}]);
     const bookNames = await c.getBooksIter({}, {}).map(b => b.name);
+    console.log("Book names: ", bookNames);
```
Comment (tnsardesai) [inline samples/test/js/paging.ts:24]: Same - is this needed?

**Review by tnsardesai — APPROVED** [review]
A few nit comments but overall looks really good 🚢

File: VERSION (hunk 1 above)
Comment (ChrisScotMartin) [reply VERSION]: 👍

File: samples/test/js/circuit_breaker.ts [hunk 3]
```diff
@@ -212,6 +207,7 @@ describe("circuit", function() {
     Promise.all(promises).then(() => {
       done();
     }).catch((err) => {
+      console.log("ERROR: ", err)
```
[the remark this answers is not in the document]
Comment (ChrisScotMartin) [reply samples/test/js/circuit_breaker.ts]: Nope, missed it when I was cleaning up my debug logs from trying to get tests to pass.

File: samples/test/js/paging.ts (hunk 2 above)
Comment (ChrisScotMartin) [reply samples/test/js/paging.ts]: Nope

**Pull request merged**